pkg/apis/openebs.io/core/v1alpha1/csivolume.go

```diff
@@ -35,19 +35,19 @@ type CSIVolume struct {
 // CSIVolumeSpec is the spec for a CStorVolume resource
 type CSIVolumeSpec struct {
 	// Volume specific info
-	Volume VolumeInfo
+	Volume VolumeInfo `json:"volume"`
 
 	// ISCSI info specific to ISCSI protocol,
 	// this is filled only if the volume type
 	// is iSCSI
-	ISCSI ISCSIInfo
+	ISCSI ISCSIInfo `json: "iscsi"`
 }
 
 // VolumeInfo contains the volume related info
 // for all types of volumes in CSIVolumeSpec
 type VolumeInfo struct {
 	// Volname is the name of the CSI volume
-	Volname string `json:"volname"`
+	Volname string `json:"name"`
```

Comment:
Should we rename the variable also to "Name" instead of Volname. As the structure variable defined above itself indicates that its a Volume.

Reply:
Done